lib/imageview360.dart

```diff
@@ -33,7 +33,6 @@ class ImageView360 extends StatefulWidget {
   final Function(int? currentImageIndex)? onImageIndexChanged;
 
   ImageView360({
-    required Key key,
```

Comment:
ImageView360 is a widget, you can keep the key property and remove the 'required' useless parameter.

Reply:
You can still keep the super(key: key)